curriculum/challenges/english/blocks/zh-a1-learn-certification-introduction/691dcd913d383a6ba27d78c8.md

```diff
@@ -0,0 +1,37 @@
+---
+id: 691dcd913d383a6ba27d78c8
+title: How the Certification Works
+challengeType: 24
+dashedName: how-the-certification-works
+lang: zh-CN
+---
+
+# --description--
+
+Welcome to the **A1 Chinese for Professionals** certification, Camper! We're thrilled to have you here. This program is all about guiding you **step by step** as you build real, practical Chinese skills from the ground up that you can use in professional settings.
+
+Here's the breakdown of how the certification is structured and how you'll learn:
+
```

Comment:
This empty line is extra.

```suggestion
```

Reply:
I saw these empty lines in the Spanish tasks, so I thought it might be some kind of convention.